spec/rspec/matchers/built_in/change_spec.rb

```diff
@@ -223,7 +223,7 @@ def ==(other)
   it "warns if passed a block using do/end instead of {}" do
     expect do
       expect {}.to change do; end
-    end.to raise_error(SyntaxError, /block passed to the `change` matcher/)
+    end.to raise_error(ArgumentError, /Block not received by the `change` matcher/)
```

Comment:
IMO it should be `ArgumentError` since `SyntaxError` means unparsable Ruby code. Also, `SytaxError` is _not_ a `StandardError`:

```bash
$ ruby -e 'p SyntaxError.ancestors'
[SyntaxError, ScriptError, Exception, Object, Kernel, BasicObject]
```

* http://ruby-doc.org/core-2.3.0/ScriptError.html
* http://ruby-doc.org/core-2.3.0/SyntaxError.html